pyspecdata/fourier/ft_shift.py

```diff
@@ -229,6 +229,7 @@ def ft_new_startpoint(self, axis, which_domain, value=None, nearest=False):
     elif which_domain == "f":
         which_domain = "freq"
         thisconditional = True
+    other_domain = tuple(set(("freq","time"))-set((which_domain,)))[0]
```

Comment:
allows us to ft a dimension other than frequency or time for example field

Reply:
That's not the point of this. We refer to the domains as frequency and time whether they are or are not

Reply:
ah so this is more compact to define the other domain vs putting it inside each if or elif independently (for example inside the if `which_doman == "t": other_domain = "f"`)